scalding-core/src/main/scala/com/twitter/scalding/source/typedtext/TypedText.scala

```diff
@@ -0,0 +1,108 @@
+package com.twitter.scalding.source.typedtext
```

Comment:
maybe this should not be in an extra level of package. Just com.twitter.scalding.source seems to be more consistent with what is already there.